### PR TITLE
Changes to launch blogs

### PR DESCRIPTION
**Pull request by gthuang**
## Description
Add workshops to the launch blogs for Azure Native v2 and Kubernetes v4, along with minor changes to help with ongoing marketing. Also small tweaks to the Azure Native workshop page.

## Checklist:

- [x] I have reviewed the [style guide](https://github.com/pulumi/pulumi-hugo/blob/master/STYLE-GUIDE.md).
- [ ] If blogging, I have reviewed the [blogging guide](https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md).
- [x] I have manually confirmed that all new links work.
- [ ] I added aliases (i.e., redirects) for all filename changes.
- [ ] If making css changes, I rebuilt the bundle.

### CODE REVIEW

File: themes/default/content/resources/azure-github-workshop/index.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ main:
...
     # Description of the webinar.
     description: |
-        When managing cloud infrastructure, it’s important to establish a build-test-release process for your team. In this session, experts from the Azure, GitHub, and Pulumi teams will show you how to use Pulumi’s Azure Native v2 provider and your favorite programming languages to stand up new projects on Azure quickly. Then, we’ll show you how Pulumi’s new Review Stacks feature creates a temporary cloud environment for every Pull Request in GitHub automatically – making it easier than ever to validate both infrastructure and app code before release.
+        When managing cloud infrastructure, it’s important to establish a build-test-release process for your team. In this session, experts from the Azure, GitHub, and Pulumi teams will show you how to use Pulumi’s [Azure Native v2 provider]() and your favorite programming languages to stand up new projects on Azure quickly. Then, we’ll show you how Pulumi’s new Review Stacks feature creates a temporary cloud environment for every Pull Request in GitHub automatically – making it easier than ever to validate both infrastructure and app code before release.
```
Comment (scottslowe) [inline themes/default/content/resources/azure-github-workshop/index.md:74]: Looks like there is a missing URL here

**Review by scottslowe — CHANGES_REQUESTED** [review]
Just a missing URL in one of the Markdown links; otherwise, looks good

File: themes/default/content/resources/azure-github-workshop/index.md (hunk 1 above)
Comment (SaraDPH) [inline themes/default/content/resources/azure-github-workshop/index.md:74]:
```suggestion
        When managing cloud infrastructure, it’s important to establish a build-test-release process for your team. In this session, experts from the Azure, GitHub, and Pulumi teams will show you how to use Pulumi’s [Azure Native v2 provider](https://www.pulumi.com/registry/packages/azure-native/) and your favorite programming languages to stand up new projects on Azure quickly. Then, we’ll show you how Pulumi’s new Review Stacks feature creates a temporary cloud environment for every Pull Request in GitHub automatically – making it easier than ever to validate both infrastructure and app code before release.
```

File: themes/default/content/blog/kubernetes-4-0-even-more-kubernetes-native/index.md [hunk 2]
```diff
@@ -23,6 +23,9 @@ Since the very earliest days of the Pulumi project, Kubernetes has been a core p
 Today, we are excited to release the next major version of our Kubernetes provider - [Pulumi Kubernetes 4.0](/registry/packages/kubernetes/).
```
Comment (SaraDPH) [inline themes/default/content/blog/kubernetes-4-0-even-more-kubernetes-native/index.md:24]:
```suggestion
We are excited to release the next major version of our Kubernetes provider - [Pulumi Kubernetes 4.0](/registry/packages/kubernetes/).

```

File: themes/default/content/blog/introducing-azure-native-v2/index.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ---
-title: "Introducing Azure Native 2.0"
+title: "Azure Native 2.0 for Infrastructure as Code on Azure"
```
Comment (SaraDPH) [inline themes/default/content/blog/introducing-azure-native-v2/index.md:2]:
```suggestion
title: "Azure Native Provider 2.0: Streamlined, Expanded, and More Powerful than Ever"
```

File: themes/default/content/blog/kubernetes-4-0-even-more-kubernetes-native/index.md [hunk 4]
```diff
@@ -215,6 +218,12 @@ As part of this work, we removed the use of the `kubectl.kubernetes.io/last-appl
...
-You can migrate your existing Pulumi Kubernetes projects to 4.0 today, or start a new project targeting 4.0 with any of the Pulumi templates.  
+You can migrate your existing Pulumi Kubernetes projects to 4.0 today, or start a new project targeting 4.0 with any of the Pulumi templates.
 
 For users migrating to 4.0, check out the [v4 Provider Migration Guide](https://www.pulumi.com/registry/packages/kubernetes/how-to-guides/v4-migration/) with full details of the additional changes in this release.
+
+## Getting Started
+
+Try Infrastructure as Code for Kubernetes by following this quick [getting started guide](/docs/clouds/kubernetes/get-started/).
+
+Join us for a free, on-demand webinar with Pulumi and the CNCF to learn how to use Pulumi's Kubernetes Provider and other open-source tools to build internal developer platforms. [Register &rarr;](https://community.cncf.io/events/details/cncf-cncf-online-programs-presents-cncf-on-demand-webinar-how-to-start-building-a-self-service-infrastructure-platform-on-kubernetes/)
```
Comment (SaraDPH) [inline themes/default/content/blog/kubernetes-4-0-even-more-kubernetes-native/index.md:229]:
```suggestion
Join us for a free, [on-demand webinar with Pulumi and the CNCF](https://community.cncf.io/events/details/cncf-cncf-online-programs-presents-cncf-on-demand-webinar-how-to-start-building-a-self-service-infrastructure-platform-on-kubernetes/) to learn how to use Pulumi's Kubernetes Provider and other open-source tools to build internal developer platforms. 
[Register &rarr;](https://community.cncf.io/events/details/cncf-cncf-online-programs-presents-cncf-on-demand-webinar-how-to-start-building-a-self-service-infrastructure-platform-on-kubernetes/)
```

**Review by SaraDPH — CHANGES_REQUESTED** [review]

**Review by scottslowe — APPROVED** [review]

**Review by SaraDPH — APPROVED** [review]

**Pull request merged**